packages/opencensus-instrumentation-http2/test/test-http2.ts

```diff
@@ -130,7 +130,7 @@ describe('Http2Plugin', () => {
     });
 
     after(() => {
-      shimmer.wrap(server.constructor.prototype, 'emit', plugin.patchEmit());
+      plugin.patchForTest('emit');
```

Comment:
Actually, I'm wondering why we need to unpatch the server to test the client in the first place (and vice versa). Is it possible to just ignore traces from the server? Whether or not we are using `patchForTest` we are still depending on class-private behavior.

Reply:
I see your point, makes sense. I removed that method and did just that, ignored the traces we don't care.